components/earnings/commons/earnings_base.lua

```diff
@@ -176,6 +186,11 @@ function Earnings.calculate(conditions, queryYear, mode, perYear, aliases, isPla
 	return totalEarnings, earningsByYear
 end
 
+---Appends query conditions depending on year and mode
```

Comment:
```suggestion
---Creates query conditions depending on year and mode
```
doesn't append does it?

Reply:
technically it gets passed player/team conditions that get appended by the common ones
but am okay with `Creates` too